app/helpers/parse/toNumber.spec.js

```diff
@@ -0,0 +1,31 @@
+const assert = require('assert')
+
+const toNumber = require('./toNumber')
+
+describe('toNumber', function () {
+    describe('when value already is number', function () {
+        it('should return miliseconds duration', function () {
+            const number = 3
+            assert.equal(number, toNumber(number))
+        })
+    })
+
+    describe('when the value is not a number', function () {
+        it('should parse simple value correctly', function () {
+            assert.equal(1, toNumber('1'))
+            assert.equal(28, toNumber('28'))
+            assert.equal(742, toNumber('742'))
+            assert.equal(1000, toNumber('1000'))
+        })
+
+        it('should parse comma value correctly', function () {
+            assert.equal(1000, toNumber('1000,00'))
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 12.](https://app.codacy.com/gh/UFABCNextOps/ufabc-next-server/pullRequest?prid=8443142)
```suggestion
      assert.equal(1000, toNumber('1000,00'))
```